Task 1/Task 1.1/Task 1.1/Task 1.1.5/Program.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using TasksClassLibrary;
+
+namespace Task_1._1._5
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Task 1.1.5\n");
+            TasksClassLibrary.SumOfMultiple.SumOfNumbers();
```

Comment:
```suggestion
            SumOfMultiple.SumOfNumbers();
```
IDE не предложила такое сокращение? По идее у тебя юзинг есть, и полное название не требуется.